src/test/java/org/folio/qm/conveter/MarcHoldingsDtoConverterTest.java

```diff
@@ -84,4 +87,13 @@ void testParsedRecordToQuickMarcJsonConversion(String parsedRecordDtoPath, Strin
     QuickMarc quickMarcJson = converter.convert(parsedRecordDto);
     mockIsEqualToObject(quickMarcJsonPath, quickMarcJson);
   }
+
+  @Test
+  void testHoldingsGeneralCharacteristicsControlFieldWrongLength(){
```

Comment:
Could you add a positive case of converting dto to qm record for 008 field in holdings?

Reply:
I have updated the test files with valid 008 field value for this test, so it might be taken as a positive case for the test ![image](https://user-images.githubusercontent.com/37537790/133448691-7de9c646-b72f-4f2a-b5b4-792671754589.png)
and the changes 
![image](https://user-images.githubusercontent.com/37537790/133449250-06559b01-84d6-4efe-a3cf-62f2d596a5ef.png)
and the corresponding quick MARC
![image](https://user-images.githubusercontent.com/37537790/133449411-a50a6acd-7b2f-4bca-b1c0-f9316dbcedfb.png)